proposals/custom-states-and-state-pseudo-class.md

```diff
@@ -120,3 +120,91 @@ This causes compatibility issues if we ever want to introduce new CSS pseudo-cla
 ### Different syntax choices for custom states, e.g. `:--foo`
     
 We are open to other suggestions. However the discussion so far seems to favor `:state(foo)`.
+
+## Security and Privacy Considerations
+
+This feature must not have security and privacy implications.  The feature works within a single document, and nothing is exposed to outside of the document.
+
+## Major open issues
+
+-   Web components should be able to expose a list of supported states to web components users.
```

Comment:
I don't think this is a major issue, IIRC something like this was discussed in TPAC and we don't really need to provide a way to solve this

Reply:
Ok, removed it.